lib/editor/selectors/timetable.js

```diff
@@ -49,6 +49,13 @@ export const getTimetableColumns = createSelector(
         key: 'tripHeadsign',
         type: 'TEXT',
         placeholder: 'Destination via Transfer Center'
+      },
+      {
+        name: 'Trip Short Name',
+        width: 200,
+        key: 'tripShortName',
+        type: 'TEXT',
+        placeholder: '801'
```

Comment:
How did you come up with the value for this?

Reply:
In light of your comment @philip-cline, my preference would be something like `Train 012 (or, just 012)`, or no placeholder at all. I'll let @landonreed break the tie.

Reply:
I think `801` is fine. I'm not really sure how that's any different from something like `012`. If we wanted to modify all of the placeholders to do something different (besides offering a single example value) that's a separate concern.

Reply:
Sure